code/game/objects/items/cards_ids.dm

```diff
@@ -1361,11 +1361,20 @@
 	return ..()
 
 /// A special variant of the classic chameleon ID card which accepts all access.
+//SKYRAT EDIT BEGIN..
+
+#define WILDCARD_LIMIT_CHAMELEON_ADVANCED list( \
+	WILDCARD_NAME_CENTCOM = list(limit = 2, usage = list()), \
+	WILDCARD_NAME_SYNDICATE = list(limit = -1, usage = list()), \
+	WILDCARD_NAME_CAPTAIN = list(limit = -1, usage = list()) \
+)
```

Comment:
Take out the CentCom access for now, we're going to add it eventually as we see fit, once we have proper policy on the Fleet, it's still too new to make this kind of change.
```suggestion
#define WILDCARD_LIMIT_CHAMELEON_ADVANCED list( \
	WILDCARD_NAME_SYNDICATE = list(limit = -1, usage = list()), \
	WILDCARD_NAME_CAPTAIN = list(limit = -1, usage = list()) \
)
```

Reply:
No. Refer to dev-discussion as to why.